projects/js-packages/connection/helpers/get-calypso-origin.ts

```diff
@@ -1,11 +1,15 @@
+import { getScriptData } from '@automattic/jetpack-script-data';
+
 /**
  * Get the Calypso origin based on the development environment.
  *
  * @returns {string} The Calypso url origin.
  */
 export default function getCalypsoOrigin() {
-	const calypsoEnv =
-		typeof window !== 'undefined' && window?.JP_CONNECTION_INITIAL_STATE?.calypsoEnv;
+	const calypsoEnv = (
```

Comment:
Should the `||` operands be inverted so we match the logic of the other changes?

Reply:
Done. Thanks